dbt/macros/generate_schema_name.sql

```diff
@@ -1,15 +1,41 @@
--- Override the default schema naming to remove the dbt-added prefix.
+-- Override the default schema naming to remove the autogenerated prefix
+-- and replace it with our own namespacing on dev and CI.
 -- See: https://docs.getdbt.com/docs/build/custom-schemas
 {% macro generate_schema_name(custom_schema_name, node) -%}
 
+    {#
+        According to the dbt docs linked above, this is required to be set by
+        the built-in macro that we are overriding, but we don't actually use it
+    #}
     {%- set default_schema = target.schema -%}
+
+    {%- if target.name == "dev" -%}
+        {%- set schema_prefix = env_var("USER") -%}
+    {%- elif target.name == "ci" -%}
+        {%- set schema_prefix = env_var("GITHUB_BASE_REF") -%}
```

Comment:
**suggestion**: Won't `GITHUB_BASE_REF` give the target of the PR i.e. `main`? I think this should be `GITHUB_HEAD_REF` instead.

Reply:
Great catch, I got these mixed up after reading the docs! Fixed in 62dc2b34473ddf9a4f63d2a1fa6a8e39368c80f8 👍🏻